distributionviewer/core/static/js/app/components/views/example-chart.js

```diff
@@ -29,6 +29,7 @@ export class ExampleChart extends React.Component {
       { x: 100, y: 86 },
     ];
 
+    /* eslint-disable camelcase */
```

Comment:
Agreed. It's only needed because the `metrics-graphics` library doesn't camel-case their property names. I wish eslint could somehow detect and ignore situations like these.